### DISCUSSION TITLE
What is the use of these files such as < Chen2020.py>

### DISCUSSION THREAD

**Discussion by TangLeiPlus** [pybamm-team/PyBaMM#2416] [Q&A] [answered]
hello everyone
I am a total beginner on Pybamm
I've got a question here. Please help.
There are some files like:
../pybamm/input/parameters/lithium_ion/Chen2020.py
this file consists of all the parameters which will be used in a simulation. I thought there might be a way for me to use this file to reset all the parameters in a simulation. But I found I can't use these files directly to change all the parameters.
I deleted all those file like Chen2020.py Ai2020.python Xu2019.py in the path above. After that, I rerun all the codes in the " example notebook", I found all the codes could run normally with no errors.
So I was wondering whether those files like "Chen2020.py" have any use.
Thanks in advance.

**Comment by rtimms** [comment, marked as answer]
These files contain the in-built parameter sets that ship with pybamm. See the [documentation ](https://github.com/pybamm-team/PyBaMM/tree/develop/examples/notebooks/parameterization)  for more info on these sets and on how to add your own parameter sets. 

[These notebooks ](https://github.com/pybamm-team/PyBaMM/tree/develop/examples/notebooks/parameterization) show how to use/edit parameter sets or define your own parameter sets directly in a notebook/script.
